PowerShellTools/DebugEngine/ScriptDebugProcess.cs

```diff
@@ -10,10 +11,22 @@ public class ScriptDebugProcess : IDebugProcess2
         private static readonly ILog Log = LogManager.GetLogger(typeof(ScriptDebugProcess));
 
         private readonly IDebugPort2 _port;
+        private string _processName;
+
+        public ScriptDebugProcess(IDebugPort2 debugPort, uint processId, string processName, string hostname) : this(debugPort)

```

Comment:
This ctor is a super set of the below ctor.  Make it call the other one, or vice versa.